test/conftest.py

```diff
@@ -137,45 +93,14 @@ def cql_test_connection(cql, request):
 def this_dc(cql):
     yield cql.execute("SELECT data_center FROM system.local").one()[0]
 
-# While the raft-based schema modifications are still experimental and only
-# optionally enabled some tests are expected to fail on Scylla without this
-# option enabled, and pass with it enabled (and also pass on Cassandra).
-# These tests should use the "fails_without_raft" fixture. When Raft mode
-# becomes the default, this fixture can be removed.
-@pytest.fixture(scope="session")
-async def check_pre_raft(cql):
-    # If not running on Scylla, return false.
-    names = [row.table_name for row in await cql.run_async("SELECT * FROM system_schema.tables WHERE keyspace_name = 'system'")]
-    if not any('scylla' in name for name in names):
-        return False
-    # In Scylla, we check Raft mode by inspecting the configuration via CQL.
-    experimental_features = list(await cql.run_async("SELECT value FROM system.config WHERE name = 'experimental_features'"))[0].value
-    return not '"raft"' in experimental_features
-
-
-@pytest.fixture(scope="function")
-async def fails_without_raft(request, check_pre_raft):
-    if check_pre_raft:
-        request.node.add_marker(pytest.mark.xfail(reason='Test expected to fail without Raft experimental feature on'))
-
-
 # "keyspace" fixture: Creates and returns a temporary keyspace to be
 # used in tests that need a keyspace. The keyspace is created with RF=1,
 # and automatically deleted at the end. We use scope="session" so that all
 # tests will reuse the same keyspace.
 @pytest.fixture(scope="session")
-async def keyspace(cql, this_dc):
+def keyspace(cql, this_dc):
```

Comment:
Nadav wasn't happy with us adding topology-only things to `test/conftest.py` and since nobody besides topology uses async, we can just keep it within `test/topology/conftest.py`. Also, Python driver doesn't have async support for connection. And the `this_dc` workaround is for running tests with Cassandra 3 (while topology tests require Scylla 5+).

I'll update the commit comment.